src/common/components/Modal/Modal.theme.ts

```diff
@@ -0,0 +1,67 @@
+import { tv, type VariantProps } from "tailwind-variants";
+
+export const modalTheme = tv({
+  slots: {
+    overlay: [
+      "z-modal",
+      "fixed",
+      "inset-0",
+      "bg-bg-base/90",
+      "backdrop-blur-sm",
+      "overflow-y-auto",
+      "p-4",
+      "sm:p-10",
+      "flex",
+      "justify-center",
+      "data-[state=closed]:animate-[dialog-overlay-hide_200ms]",
+      "data-[state=open]:animate-[dialog-overlay-show_200ms]",
+    ],
+    close: [
+      "absolute",
+      "right-6",
+      "top-5",
+      "sm:right-8",
+      "sm:top-[30px]",
+      "text-text-placeholder",
+      "z-[1]",
+    ],
+    header: ["flex", "flex-col", "gap-2", "items-start", "px-6 ", "sm:px-8"],
+    headerDescription: ["text-sm", "text-text-em-low"],
+    footer: ["flex", "flex-col", "gap-4", "px-6 md:px-8"],
+    content: [
+      "m-auto",
+      "relative",
+      "flex",
+      "flex-col",
+      "gap-y-6",
+      "py-6",
+      "sm:py-8",
+      "overflow-hidden",
```

Comment:
suggestion:
shall we make the modal content scrollable as the default behavior instead of scrolling the whole modal? this way the header and footer will be visible at all times

Reply:
Oh okok sounds good. I added a new variant to allow overflow inside, or outside via a prop `overflow='inside' | 'outside'`.

https://github.com/AfterClass-io/afterclass.io-v2/pull/59/commits/aa04afee2eb3e54537a230ec098239dc9e6c6309

https://github.com/AfterClass-io/afterclass.io-v2/assets/6003064/8ce2b568-444e-4fce-82f7-c2b39e80ae8b